.gitignore

```diff
@@ -8,3 +8,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+*~
+\#*
```

Comment:
Were these meant to be included?

Reply:
emacs temp file patterns. I can strip them out, but they were useful to have while developing.

Reply:
@seanmonstar any preference here? I lean towards larger .gitignores, just to avoid accidental commits into a repo.

Reply:
I tend to think these files should be scoped to the specific project. It's always possible for any user to have a `git config --global core.excludesfile`, so you can prevent those files on your computer always.

Reply:
Thanks. I wasn't aware of that feature.